inc/fields.php

```diff
@@ -123,10 +123,23 @@ function civil_first_fleet_fm_submenu_newsroom_settings() {
 					[
 						'label' => __( 'Analytics', 'civil-first-fleet' ),
 						'children' => [
-							'ga_property_code' => new Fieldmanager_TextField(
+							'properties' => new Fieldmanager_Group(
```

Comment:
Since you are changing the data structure here, is there a concern that this will break existing data?